tests/components/mqtt/test_state_vacuum.py

```diff
@@ -57,80 +53,90 @@
 
 async def test_default_supported_features(hass, mqtt_mock):
     """Test that the correct supported features."""
-    assert await async_setup_component(
-        hass, vacuum.DOMAIN, {vacuum.DOMAIN: DEFAULT_CONFIG}
-    )
+    assert await async_setup_component(hass, vacuum.DOMAIN,
+                                       {vacuum.DOMAIN: DEFAULT_CONFIG})
     entity = hass.states.get("vacuum.mqtttest")
-    entity_features = entity.attributes.get(mqttvacuum.CONF_SUPPORTED_FEATURES, 0)
-    assert sorted(services_to_strings(entity_features, SERVICE_TO_STRING)) == sorted(
-        ["start", "stop", "return_home", "battery", "status", "clean_spot"]
-    )
+    entity_features = entity.attributes.get(mqttvacuum.CONF_SUPPORTED_FEATURES,
+                                            0)
+    assert sorted(services_to_strings(entity_features,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)